flow/ThreadPrimitives.h

```diff
@@ -59,10 +59,10 @@ class ThreadSpinLock {
 	}
 	void enter() {
 		while (isLocked.test_and_set(std::memory_order_acquire))
-#ifndef __aarch64__
```

Comment:
I'm wondering if this makes Mac build fail?

Reply:
It shouldn't. `_mm_pause()` for Intel has been there for a long time.
`isb` instruction should be available on ARMv7 and later, I think.